src/Security/LoginFormAuthenticator.php

```diff
@@ -154,6 +156,11 @@ public function onAuthenticationSuccess(Request $request, TokenInterface $token,
 
     public function onAuthenticationFailure(Request $request, AuthenticationException $exception): RedirectResponse
     {
+        // Don't reveal a UsernameNotFound exception.
+        if ($exception instanceof BadCredentialsException || $exception instanceof UsernameNotFoundException) {
```

Comment:
Isn't it enough to do this, since in the other case it already is `BadCredentialsException`? 

```suggestion
        if ($exception instanceof UsernameNotFoundException) {
```